### PR TITLE
Ability to Run as python script + Standalone app

### PR DESCRIPTION
**Pull request by narfster**
Hi,
Added 
* using pyinstaller and batch file we can now use cyflash.exe as a portable app
* ability to run as script directly from bootload.py
* changes in readme.md

thanks

### CODE REVIEW

File: cyflash/bootload.py [hunk 1]
```diff
@@ -8,9 +8,10 @@
 
 from builtins import input
 
-from . import cyacd
-from . import protocol
-
+#from . import cyacd
```
Comment (jsiverskog) [inline cyflash/bootload.py]: remove this if it shouldn't be there anymore

**Comment by jsiverskog** [comment]
thanks for your merge request.

i don't think the stuff in `build_app_win` is general enough to be in this repository.

also, the readme commits should perhaps be squashed, and the merge commit doesn't seem necessary.

**Comment by jsiverskog** [comment]
also, your changes to `bootload.py` breaks python 3 compatibility.

**Comment by narfster** [comment]
hi,
1.  changes in  `bootload.py` were done with python 3.7, it works fine on python 3 i think..
**objective** - to run the python script without generating the setup.py install script.
how should it be done then?

2. won't it be nice to generate an executable cyflash.exe free of all these dependencies and run in on any platform PC? isn't that a general concept? 
your know the exe located in python script folder is not portable at all.

for my use - I won't want to install python on each windows PC just to use cyflash.exe.

File: cyflash/bootload.py (hunk 1 above)
Comment (narfster) [reply cyflash/bootload.py]: i fix this after i understand a better solution...

Comment (narfster) [reply cyflash/bootload.py]: restored.

**Comment by jsiverskog** [comment]
instead of `cyflash_run.py` perhaps you can use a `virtualenv` + `python setup.py develop`

**Comment by jsiverskog** [comment]
generating exe files out of python packages is something that (in my opinion) should be kept out of the package repo unless it's a base feature of the package. especially when it contains hard-coded paths like in your case.

**Comment by narfster** [comment]
I'm not familiar with `virtualenv`

anyhow i'll keep this extra stuff on my fork

thanks for your contribution!!

Dan

**Pull request closed without merging**